integr8ly/installation
# Mobile monitoring templates

## Verification Steps

If you want to run the playbook:

* checkout the branch
* run the following commands against an existing Integreatly cluster:
```
ansible-playbook -i inventories/hosts playbooks/upgrades/upgrade.yaml -e mobile_security_service=true -e eval_mobile_security_service_namespace=test-mss
ansible-playbook -i inventories/hosts playbooks/install_middleware_monitoring.yml -e mobile_security_service=true -e eval_mobile_security_service_namespace=test-mss
```
* You may see some errors but as long as they are not related to the mobile security service, then it's ok. 
* When both are finished, login to the Grafana dashboard of the `Managed Service Monitoring` project, and you should see dashboards for both MSS service and its operator.

If you don't want to run the playbook, you can check this cluster where I have already ran the playbook: https://grafana-route-middleware-monitoring.apps.mobile-6cab.openshiftworkshop.com/?orgId=1. Check the dashboards for MSS service and its operator. There won't be any data for the moment as Prometheus itself is not working properly.
